charts/posthog/templates/pgbouncer-deployment.yaml

```diff
@@ -82,13 +78,26 @@ spec:
         readinessProbe:
           tcpSocket:
             port: 6543
-          initialDelaySeconds: 5
-          periodSeconds: 10
+          failureThreshold: {{ .Values.pgbouncer.readinessProbe.failureThreshold }}
+          initialDelaySeconds: {{ .Values.pgbouncer.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.pgbouncer.readinessProbe.periodSeconds }}
+          successThreshold: {{ .Values.pgbouncer.readinessProbe.successThreshold }}
+          timeoutSeconds: {{ .Values.pgbouncer.readinessProbe.timeoutSeconds }}
+
         livenessProbe:
           tcpSocket:
             port: 6543
-          initialDelaySeconds: 15
-          periodSeconds: 20
+          failureThreshold: {{ .Values.pgbouncer.livenessProbe.failureThreshold }}
+          initialDelaySeconds: {{ .Values.pgbouncer.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.pgbouncer.livenessProbe.periodSeconds }}
+          successThreshold: {{ .Values.pgbouncer.livenessProbe.successThreshold }}
+          timeoutSeconds: {{ .Values.pgbouncer.livenessProbe.timeoutSeconds }}
+
+        lifecycle:
```

Comment:
🥳

Reply:
What's the default behaviour?

Reply:
https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/

Reply:
me gusta 🥳